src/main/proto/wfa/virtual_people/common/event.proto

```diff
@@ -30,6 +30,9 @@ message EventId {
   // The publisher specific event id. Unique within all logs of the
   // publisher.
   optional string id = 2;
+
+  // The fingerprint of id. Only used in Labeler.
```

Comment:
Maybe "Only set and used in Labeler." to make it clear this field is not set in input events.

Reply:
Done.